### PR TITLE
fix: address vulnerabilities, bump packages

### PR DESCRIPTION
**Pull request by davidma415**
Bump packages to address prototype pollution vulnerabilities

### CODE REVIEW

File: build/rollup.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ import json from "rollup-plugin-json"
 import commonjs from "rollup-plugin-commonjs"
 import resolve from "rollup-plugin-node-resolve"
 import filesize from "rollup-plugin-filesize"
-import uglify from "rollup-plugin-uglify"
+import { uglify } from "rollup-plugin-uglify";
```
Comment (andrewwylde) [inline build/rollup.config.js]:
```suggestion
import { uglify } from "rollup-plugin-uglify"
```

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -13,7 +13,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [12.x, 14.x]
+        node-version: [14.x]
```
Comment (andrewwylde) [inline .github/workflows/test.yml]: if we go down to just one version, should we just set at top level?

Alternatively could add `16` and `18` and verify it builds.

Comment (davidma415) [reply .github/workflows/test.yml]: Added a few versions

**Review by andrewwylde — APPROVED** [review]
I do love to see it. 
image

**Pull request merged**